src/FluentValidation/Internal/Extensions.cs

```diff
@@ -109,8 +109,12 @@ public static class Extensions {
 		}
 
 		/// <summary>
-		/// Splits pascal case, so "FooBar" would become "Foo Bar"
+		/// Splits pascal case, so "FooBar" would become "Foo Bar".  
 		/// </summary>
+		/// <remarks>
+		/// Pascal case strings with periods delimiting the upper case letters,
+		/// such as "Address.Line1", will have the periods removed.  
+		/// </remarks>
 		public static string SplitPascalCase(this string input) {

```

Comment:
The removal of the delimiting periods for only pascal cased strings seems like it might not belong in a `SplitPascalCase` method. Maybe we could create a new method that leverages this one, or would be able to be an extension method for `PropertyName`-esque strings. 
That said, maybe this is fine as this is already only ever used for display of different property names.